src/envelope.rs

```diff
@@ -48,11 +50,44 @@ pub trait MessageEnvelope: Send {
     ) -> BoxFuture<'a, ()>;
 }
 
+#[cfg(feature = "with-tracing-0_1")]
+#[derive(Clone)]
+struct Instrumentation {
+    parent: Span,
+    _waiting_for_actor: Span,
+}
+
+#[cfg(feature = "with-tracing-0_1")]
+impl Instrumentation {
+    fn new<A: Actor, M>() -> Self {
+        let parent = debug_span!(
+            parent: Span::current(),
+            "xtra actor request",
```

Comment:
Is naming of spans with spaces idiomatic? I've not worked with traces much but I could imagine that simple processing on the commandline could be unnecessarily complicated with this having spaces in it.

Reply:
I've only used Jaeger as a trace explorer, where it does not matter. Maybe we should change it to _s to be safe